gandalf.js

```diff
@@ -1,68 +1,61 @@
 /**
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?